CI.md

```diff
@@ -0,0 +1,184 @@
+# CI information for google-cloud-ruby
+
+This document describes how continuous integration is configured for this repository. It documents the current setup, what runs when, how to modify the configuration, and what tools exist to run jobs locally.
+
+## Overview
+
+Continuous integration refers to the various tests run automatically against the source in this repository under different conditions. These tests can be enumerated in a matrix with the following five dimensions:
+
+1.  **Test trigger**
+
+    Continuous integration runs in response to four types of events:
+
+    * **Presubmit**: Tests may run to validate code in a pull request before it is accepted into the main code base.
+    * **Continuous**: Tests may run when new code is merged into the main branch.
+    * **Nightly**: Tests may run each night on a schedule.
+    * **Manual**: Some tests may be triggered manually.
+
+2.  **Test type**
+
+    Several types of tests are defined, to cover different things:
+
+    * **Unit tests**: Small, fast tests that check the code itself.
+    * **Rubocop test**: The [Rubocop](https://rubocop.org/) style checker and linter.
+    * **Build test**: Runs a build of the gem.
+    * **Yardoc test**: Runs the [YARD](https://yardoc.org/) documentation generation.
+    * **Linkinator test**: Runs the [Linkinator](https://github.com/JustinBeckwith/linkinator) tool to find broken documentation links.
+    * **Acceptance tests**: A small number of slower tests for a client that runs against the actual backend service.
+    * **Sample tests**: Tests of code samples, usually run against the actual backend service.
+
+3.  **OS and architecture platform**
+
+    We support several operating systems: Linux, MacOS, and Windows, mostly on the 64-bit Intel architecture, but this is being extended to include ARM. Some tests are run against a limited subset of this matrix, while others cover a broader set.
+
+4.  **Ruby version**
+
+    Tests may run against one or multiple Ruby versions. We generally support four Ruby versions at any given time, but we may run some tests against only a subset of those.
```

Comment:
Provide a link or explanation of where to see which Ruby versions are currently tested?